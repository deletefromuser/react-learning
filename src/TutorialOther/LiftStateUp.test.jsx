import { act, create } from 'react-test-renderer';
import { BoilingVerdict } from './LiftStateUp';

describe("BoilingVerdict", () => {
    let root;

    it('not boils', () => {
        // render the component
        let root;
        act(() => {
            root = create(<BoilingVerdict celsius={1} />)
        });

        expect(root.toJSON()).toMatchSnapshot();
    });

    it('boils', () => {
        // render the component
        let root;
        act(() => {
            root = create(<BoilingVerdict celsius={101} />)
        });

        expect(root.toJSON()).toMatchSnapshot();
    });
});
