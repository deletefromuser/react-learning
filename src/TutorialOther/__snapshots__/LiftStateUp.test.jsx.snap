// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoilingVerdict boils 1`] = `
<p>
  The water would boil.
</p>
`;

exports[`BoilingVerdict not boils 1`] = `
<p>
  The water would not boil.
</p>
`;
